osu.Game/Screens/Play/GameplayMenuOverlay.cs

```diff
@@ -217,6 +227,21 @@ public void OnReleased(KeyBindingReleaseEvent<GlobalAction> e)
         {
         }
 
+        private bool useButtonKeybind(string id)
```

Comment:
Even if we wanted this hotkey functionality (which I'm not sure if we do, and you would have done well if you asked beforehand, as we generally suggest asking before PRing anything), this magic stringy API cannot exist. `GameplayMenuOverlay` doesn't even have any buttons in it by default, so why should it be handling hotkeys via a magic name string for its derived classes? It's all backwards.

The proper way to do this would be to add an `InputKey?` optional param to `AddButton()` and hook things up through there.

Reply:
That said, I'm not sure how contextual we want to add global binds like this (ie. when they only work at one very specific screen).

To an extent I agree with something similar the underlined keys in windows menus, hinting at the key to activate the dialog option. Probably something in addition to the (very hidden) 1,2,3 button shortcuts in dialogs.

Doesn't help that pause overlay is a very custom one right now.

Reply:
> To an extent I agree with something similar the underlined keys in windows menus, hinting at the key to activate the dialog option.

For this to work best, we'd also need to pass down the keycode (in addition to the current scancode-`osuTK.Key`).

Reply:
True. I think that these shortcuts wouldn't really make sense for these small menus, since most of the players already have a muscle memory of either moving their hands over to the enter and arrow keys, using the mouse or having a mod like Sudden Death or Perfect enabled. I'll let you choose if I should add the changes bdach requested or if you close this pull request. I could also just add an issue and close this pull request in case this feature is wanted in the future

Reply:
I would lean towards closing this PR. A better direction may involve porting [support for number hotkeys](https://github.com/ppy/osu/blob/4bc26dbb487241e2bbae73751dbe9e93a4e427da/osu.Game/Overlays/Dialog/PopupDialog.cs#L231-L251) across from `PopupDialog` to the gameplay menus.